cmake/release_dependencies.cmake

```diff
@@ -22,52 +27,35 @@ FetchContent_Declare( hana-adapter
 
 FetchContent_Declare( header-utilities
     GIT_REPOSITORY  https://github.com/njoy/header-utilities
-    GIT_TAG         04f988c2b576630ef10fc0c3838c915f4a6e15d6
-    )
-
-FetchContent_Declare( ENDFtk
-    GIT_REPOSITORY  https://github.com/njoy/ENDFtk
-    GIT_TAG         1217062c298560824cb8f239465b3293df178deb
-    )
-
-FetchContent_Declare( catch-adapter
-    GIT_REPOSITORY  https://github.com/njoy/catch-adapter
-    GIT_TAG         fb84b82ebf7a4789aa43cea560680cf745c6ee4f
+    GIT_TAG         cc2610fee15e255c151e8e22aca1e8b3d1a96b39
     )
 
 FetchContent_Declare( Log
     GIT_REPOSITORY  https://github.com/njoy/Log
-    GIT_TAG         e0f371dfef22b655bbd52e8783d037e6c594824a
-    )
-
-FetchContent_Declare( fmt-adapter
-    GIT_REPOSITORY  http://github.com/njoy/fmt-adapter
-    GIT_TAG         a6db6c8cf9534b028eb3dea4549934f7ae6abf9f
+    GIT_TAG         52962b7796afe37ef1d8f7edb4bf9ecb1b868d15
     )
 
-FetchContent_Declare( spdlog-adapter
-    GIT_REPOSITORY  http://github.com/njoy/spdlog-adapter
-    GIT_TAG         39cd17919b2be662a45b9c33342b5a5f4f1feb4c
+FetchContent_Declare( range-v3-adapter
+    GIT_REPOSITORY  https://github.com/njoy/range-v3-adapter
+    GIT_TAG         252679d4737c8f755d87c0e1eed6c37394a2ec59
     )
 
-FetchContent_Declare( disco
-    GIT_REPOSITORY  https://github.com/njoy/disco
-    GIT_TAG         178a03cf4a571fee028b008ffac85e20f6059ade
+FetchContent_Declare( spdlog
+    GIT_REPOSITORY  https://github.com/gabime/spdlog
+    GIT_TAG         a51b4856377a71f81b6d74b9af459305c4c644f8
     )
```

Comment:
The `spdlog` dependency requires an extra line to squash its unit tests.  I'll get that updated in devtools soon.